editor/plugins/theme_editor_plugin.cpp

```diff
@@ -2888,7 +2925,7 @@ void ThemeEditor::_theme_save_button_cbk(bool p_save_as) {
 }
 
 void ThemeEditor::_theme_edit_button_cbk() {
-	theme_edit_dialog->popup_centered(Size2(850, 760) * EDSCALE);
+	theme_edit_dialog->popup_centered(Size2(850, 700) * EDSCALE);
```

Comment:
Does a 700 pixel-tall popup still fit on a 1366×768 monitor with the Godot editor maximized, but not fullscreen? Assuming it's run on Windows with a non-autohiding task bar and a visible window border, that is.

Reply:
It should, I think. The user who reported it not fitting shared a screenshot where it was almost okay (but they also had a weird combination of Windows and Godot Editor scaling, so I'm not entirely sure how representative it is).